sdk/cosmos/integration-matrix.json

```diff
@@ -1,12 +1,26 @@
 {
   "matrix": {
     "Cosmos": {
-      "Session_Integration": {
+      "Session_Integration_8": {
+        "ArmTemplateParameters": "@{ enableMultipleWriteLocations = $false; defaultConsistencyLevel = 'Session' }",
+        "ProfileFlag": "-P integration-test-azure",
+        "Pool": "azsdk-pool-mms-ubuntu-2004-general",
+        "OSVmImage": "MMSUbuntu2004",
+        "JavaTestVersion": "1.8"
+      },
+      "Session_Integration_11": {
         "ArmTemplateParameters": "@{ enableMultipleWriteLocations = $false; defaultConsistencyLevel = 'Session' }",
         "ProfileFlag": "-P integration-test-azure",
         "Pool": "azsdk-pool-mms-ubuntu-2004-general",
         "OSVmImage": "MMSUbuntu2004",
         "JavaTestVersion": "1.11"
+      },
+      "Session_Integration_17": {
+        "ArmTemplateParameters": "@{ enableMultipleWriteLocations = $false; defaultConsistencyLevel = 'Session' }",
+        "ProfileFlag": "-P integration-test-azure",
+        "Pool": "azsdk-pool-mms-ubuntu-2004-general",
+        "OSVmImage": "MMSUbuntu2004",
+        "JavaTestVersion": "1.17"
       }
     }
   }
```

Comment:
This matrix will be easier to read and maintain if you de-duplicate the config entries:

```
{
  "matrix": {
    "cosmos": {
      "Session_Integration": {
        "ArmTemplateParameters": "@{ enableMultipleWriteLocations = $false; defaultConsistencyLevel = 'Session' }",
        "ProfileFlag": "-P integration-test-azure",
        "Pool": "azsdk-pool-mms-ubuntu-2004-general",
        "OSVmImage": "MMSUbuntu2004"
      }
    },
    "JavaTestVersion": ["1.8", "1.11", "1.17"]
  }
}
```